bitnami/tomcat/templates/statefulset.yaml

```diff
@@ -27,20 +27,23 @@ spec:
       annotations: {{- include "common.tplvalues.render" (dict "value" .Values.podAnnotations "context" $) | nindent 8 }}
       {{- end }}
     spec: {{- include "tomcat.pod" . | nindent 6 }}
-  {{- if and .Values.persistence.enabled }}
   volumeClaimTemplates:
   - metadata:
       name: data
     spec:
-      accessModes: {{ .Values.persistence.accessModes }}
-      storageClassName: {{ .Values.persistence.storageClassName }}
+      accessModes:
+      {{- range .Values.persistence.accessModes }}
+      - {{ . | quote }}
+      {{- end }}
       resources:
         requests:
-          storage: {{ .Values.persistence.size }}
+          storage: {{ .Values.persistence.size | quote }}
+      {{- include "common.storage.class" (dict "persistence" .Values.persistence "global" .Values.global) | nindent 6 }}
+      {{- with .Values.persistence.selectorLabels }}
       selector:
-        matchLabels: {{- include "common.labels.matchLabels" . | nindent 10 }}
+        matchLabels: {{- toYaml . | indent 10 }}
```

Comment:
Shouldn't this be using `nindent` as well? If not it would include the indent in the same label as `matchLabels:`

Reply:
Thank you I fixed it